adapter/internal/discovery/xds/server.go

```diff
@@ -296,6 +296,7 @@ func UpdateAPI(vHost string, apiProject mgw.ProjectAPI, environments []string) (
 			}
 		}
 		mgwSwagger.SanitizeAPISecurity(isYamlAPIKey, isYamlOauth)
+		mgwSwagger.SetOperationPolicies(apiYaml.Operations)
```

Comment:
Since we also have throttling policies, shall use the term "mediation policy" when ever possible?

Reply:
Resolving the comment after referring to the doc here. https://docs.google.com/document/d/1pVuhTCD8ill6_xWmPb0KAz782K87oxtGI-0MOsagyJw/edit#heading=h.2i6mc7s9o9r9